README.md

```diff
@@ -4,7 +4,7 @@
 
 ## Buildkite Elastic CI Stack for AWS
 
-[Buildkite](https://buildkite.com/) is a platform for running fast, secure, and scalable continuous integration pipelines on your own infrastructure.
+[Buildkite](https://buildkite.com/) and Pipelines is a platform for running fast, secure, and scalable continuous integration pipelines on your own infrastructure.
```

Comment:
I know Buildkite is traditionally how Buildkite has referred to its Pipelines product in the past, but since Pipelines is a product in its own right, as is Test Engine and Package Registries too, it's better to clarify which of these products the Elastic CI Stack relates to. Since Buildkite provides the platform that runs these 3 products, I feel this addition is a good compromise that fits in with the existing wording without having to re-factor this entire sentence.

Reply:
@gilesgas I'm against these changes as the first version "Buildkite and Pipelines" would imply that Buildkite is a product in the line of products like Pipelines. The second suggestion "[Buildkite](https://buildkite.com/)'s Pipelines provides a platform for running fast, secure, and scalable continuous integration pipelines on your own infrastructure." is closer to the way things are but creates redundancy and mentiones pipelines twice.
I'd rather not change this part of the Readme at all.

Reply:
I removed the addition of "and Pipelines".